.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -99,7 +99,6 @@ Additional information can be found in the [release team lead handbook](https://
   - [ ] Add Role Leads and Role shadows to members of `release-team`
   - [ ] Add Role shadows and Lead shadows to members of `release-team-shadows`
   - [ ] Add EA to manager of release-team-shadows, if EA is owner of `release-team-shadows` already then add Lead to manager of `release-team-shadows`
-  - [ ] Add Bug Triage Lead and shadows to `k8s-infra-rbac-triageparty-release`
```

Comment:
@Vyom-Yadav — can you create an issue to clean up Triage Party related things?

I'm not sure that we use it anymore, but I also want to make sure we don't leave any infra dangling if we don't.

Reply:
ref: https://github.com/kubernetes/sig-release/issues/2660
/hold cancel